### PR TITLE
fix: fix destroy method

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#403

### CODE REVIEW

File: src/DOMRenderer.ts [hunk 1]
```diff
@@ -246,7 +251,9 @@ export default class DOMRenderer {
 		for (property in this._orgStyle) {
 			(this[container ? "view" : "container"] as HTMLElement).style[property] = this._orgStyle[property]!;
 		}
-		container && this.container.parentNode!.removeChild(this.container);
+		if (container === true) {
```
Comment (WoodNeck) [inline src/DOMRenderer.ts:254]: An option named `container` has a type of boolean?
That looks kinda weird.

**Review by WoodNeck — APPROVED** [review]
LGTM

File: src/DOMRenderer.ts (hunk 1 above)
Comment (daybrush) [reply src/DOMRenderer.ts:254]: The container options are boolean and HTMLElement.

If container is true, DOM Renderer automatically creates it.

The name will be clearly changed in v4.

File: src/DOMRenderer.ts [hunk 2]
```diff
@@ -246,7 +251,9 @@ export default class DOMRenderer {
...
+			this.container.parentNode!.removeChild(this.container);
+		}
 	}
 	private _init(el: HTMLElement | IJQuery | string) {
```
Comment (mixed) [inline src/DOMRenderer.ts:258]: It's not related to this PR, but I think it's necessary to think about whether I should still get the jQuery type.

**Review by mixed — APPROVED** [review]
LGTM

File: src/DOMRenderer.ts (hunk 2 above)
Comment (daybrush) [reply src/DOMRenderer.ts:258]: @mixed 

Because it is an unofficial type, it was used only by the framework.

**Pull request merged**